Common/UnitDefinitions/Torque.json

```diff
@@ -118,8 +118,8 @@
     {
       "SingularName": "GramForceMeter",
       "PluralName": "GramForceMeters",
-      "FromUnitToBaseFunc": "x*0.009806650199607",
-      "FromBaseToUnitFunc": "x*0.000101971619222",
+      "FromUnitToBaseFunc": "x*0.00980665",
+      "FromBaseToUnitFunc": "x*101.9716213",
```

Comment:
Standard gravity is defined as `9.80665`.
I believe this can be changed for this and the other units to ensure the full precision is kept:

```
      // GramForceMeter
      "FromUnitToBaseFunc": "x*9.80665e-3",
      "FromBaseToUnitFunc": "x/9.80665e-3",

      // GramForceMillimeter
      "FromUnitToBaseFunc": "x*9.80665e-6",
      "FromBaseToUnitFunc": "x/9.80665e-6",
```

I noticed KilogramForceMeter is defined as `9.80665019960652`, but that doesn't seem right to me? Out of scope for this PR, but I think it's odd that they are slightly different from the standard definition of the constant.

Reply:
Yes, I also found it strange that "FromBaseToUnit" should be the division method, but I saw that KilogramForceMeter is like this, so I wrote it as it is